src/components/DayPickerNavigation.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { PropTypes } from 'react';
+import cx from 'classnames';
+
+import LeftArrow from '../svg/arrow-left.svg';
+import RightArrow from '../svg/arrow-right.svg';
+import ChevronUp from '../svg/chevron-up.svg';
+import ChevronDown from '../svg/chevron-down.svg';
+
+const propTypes = {
+  navPrev: PropTypes.node,
+  navNext: PropTypes.node,
+  isVertical: PropTypes.bool,
+
+  handlePrevMonthClick: PropTypes.func,
+  handleNextMonthClick: PropTypes.func,
+};
+const defaultProps = {
+  navPrev: null,
+  navNext: null,
+  isVertical: false,
+
+  handleNextMonthClick() {},
+  handlePrevMonthClick() {},
+};
+
+export default function DayPickerNavigation(props) {
+  const {
+    navPrev,
+    navNext,
+    isVertical,
+    handlePrevMonthClick,
+    handleNextMonthClick,
+  } = props;
+
+  let navPrevIcon = navPrev;
+  let navNextIcon = navNext;
+  let isDefaualtNavPrev = false;
+  let isDefaultNavNext = false;
+  if (!navPrevIcon) {
+    isDefaualtNavPrev = true;
+    navPrevIcon = isVertical ? <ChevronUp /> : <LeftArrow />;
+  }
+  if (!navNextIcon) {
+    isDefaultNavNext = true;
+    navNextIcon = isVertical ? <ChevronDown /> : <RightArrow />;
+  }
+
+  const prevClassNames = cx('DayPicker__nav--prev', {
```

Comment:
Since we've got the nice new structure here, let's rename the classes to match. I also think the prev/nav make more sense as elements rather than modifiers so maybe we'd have:
```
DayPickerNavigation
DayPickerNavigation--horizontal
DayPickerNavigation--vertical
-- DayPickerNavigation__prev
-- DayPickerNavigation__prev--default
-- DayPickerNavigation__next
-- DayPickerNavigation__next--default
```